### DISCUSSION TITLE
Feature Request: Premade In-game messages attached to moderation action

### DISCUSSION THREAD

**Discussion by CrankyLandlord** [LBPUnion/ProjectLighthouse#977] [Ideas]
If you press a button like "delete level" "wipe earth" or "Forcibly lock level" there should be automatic in-game messages that moderators can choose from to send out to moderated individuals, sort of how admins can send out but pre-made. For example say your a moderator and you click delete on a level, instead of a y/n confirmation you get a few buttons that send specific in game messages to the user, could still include an option for a generic one 'Level X has been deleted by a moderator' but there'd be more specific ones like 'Level X has been deleted because it contains 'stolen assets', 'Level X has been deleted because it contains disruptive/gamebreaking assets', 'Level X has been deleted because it is a duplicate of a previously uploaded level' so on and so forth. I'm aware that different instances of Lighthouse may have unique rules but I think if there's a rule a particular instance doesn't enforce, moderators could simply choose not to use that correlating message.

Speaking particularly of something I notice on Beacon, reuploads being taken down then put back up is a pretty consistent issue. Often people will post identical duplicate levels, they'll be deleted by a mod, and then a few days or weeks later they reupload it most likely due to not knowing the rules and from not checking the reason for moderation on the site. Because of this its often wise/necessary for moderators to leave in-game comments explaining why levels were taken down, however this is not only informal/unprofessional/an extra step for mods, it can sometimes lead to unnesscarily personal quarrels over basic housekeeping, depending on the moderated's temperament

**Comment by sudokoko** [comment]
Hello there, thank you for opening this discussion!

Seeing as this is a feature request, I'm going to go ahead and create a new issue from this discussion shortly.

You'll be credited as the original author of the suggestion in the issue.

**Comment by sudokoko** [comment]
This discussion has been transferred to issue #979.
